version_control.go

```diff
@@ -220,13 +224,25 @@ func (c *VersionCache) UpdateBinary(ctx context.Context, binary string) (bool, e
 	}
 
 	isCustomURL := strings.HasPrefix(verData.Version, "customURL+")
+	isFileURL := strings.HasPrefix(verData.URL, "file://")
 	shasum, err := utils.GetFileSum(verData.UnpackedPath)
 	if err == nil {
 		goodBytes = bytes.Equal(shasum, verData.UnpackedSHA)
 	} else if verData.UnpackedPath != "" { // custom file:// URLs with have an empty unpacked path; no need to warn
 		c.logger.Warnw("Could not calculate shasum", "path", verData.UnpackedPath, "error", err)
 	}
 
+	prevLastModified := verData.LastModified
+	var lastModified time.Time
+	var lastModifiedChanged bool
+	if isCustomURL && !isFileURL && time.Since(verData.LastModifiedCheck) > 2*time.Minute {
```

Comment:
> 2 * time.Minute

[q] How often does `UpdateBinary` run? Do we need the extra 2 minute interval on top of whatever the `UpdateBinary` interval is?